src/core/providers/ept/qgseptdataitems.cpp

```diff
@@ -29,6 +29,7 @@ QgsEptLayerItem::QgsEptLayerItem( QgsDataItem *parent,
   : QgsLayerItem( parent, name, path, uri, QgsLayerItem::PointCloud, QStringLiteral( "ept" ) )
 {
   mToolTip = uri;
+  mIconName = QStringLiteral( "mIconEntwineLayer.svg" );
```

Comment:
Not sure how others feel about icons, I would probably use a single icon for point cloud layers... and do not make difference whether it is EPT or PDAL - for users that's not really important...

Reply:
Yeah, I agree with this. (Just like we don't use a different icon for gpkg vs shp)

Reply:
On this same note, I think we should only have a single Point Cloud source select widget. It's just clutter in the data source manager if we separate EPT/PDAL. I can fix this post merge though!

Reply:
ok, lets do after merge (if needed -> see comment below)